src/ost_cli/main.py

```diff
@@ -0,0 +1,140 @@
+import os
+import sys
+import getpass
+import logging
+import argparse
+from typing import List
+from pathlib import Path
+
+from opensubtitlescom import OpenSubtitles, FileUtils, Config
+
+from .table import dicts_to_pt
+
+logger = logging.getLogger(__name__)
+
+
+def set_credentials(args: argparse.Namespace):
+    """Set the username and password in the config file
+    """
+    cfg = Config(args.config)
+    entered_appname = input("Enter your app name (leave blank to keep existing): ")
+    if entered_appname.strip():
+        cfg.app_name = entered_appname
+    entered_api_key = input("Enter your app key (leave blank to keep existing): ")
+    if entered_api_key.strip():
+        cfg.api_key = entered_api_key
```

Comment:
My understanding from the API docs is that `app name` and `app key` are expected to be hard-coded into the app -- asking the user to input these things makes it seem like every user would need to register their own app, even though they are all using the same app?

(Specifically - my understanding is that the library code should accept app name and key as parameters, and the CLI should have app name and key hard coded which it passes to the library)

Reply:
I didn't fully understand. 

User-Agent HTTP request header must be always set to your application/script/project name with version - every user would need to use their own app name and version.

An API key is a token that you provide when making API call - every user would need to use their own token.

Reply:
My reading was that every _application_ has its own app name and token, every _user_ has their own username and password. Are you saying that if Alice, Bob, and Charlie each want to use the MySubtitleDownloader app, then all of them need to sign up as opensubtitles.com developers and register three different applications?

Reply:
So I emailed the opensubtitles.com devs to ask for clarity:

> API key should be hardcoded in application - so we know it is your application.
> Users should provide their username and password, so we know about users.